cmd/gateway-gcs.go

```diff
@@ -192,12 +192,16 @@ func gcsToObjectError(err error, params ...string) error {
 
 	bucket := ""
 	object := ""
+	uploadID := ""
 	if len(params) >= 1 {
 		bucket = params[0]
 	}
 	if len(params) == 2 {
 		object = params[1]
```

Comment:
`bucket` and `object` values aren't used when we return error of type `InvalidUpload`, which is the only place where we use `uploadID`. This is intentionally left as is.

Reply:
This `if` block needs to be changed to `if len(params) >=2` or `object` will be empty when len(params) == 3.